.circleci/config.yml

```diff
@@ -0,0 +1,96 @@
+version: 2.1
+
+executors:
+  default:
+    working_directory: ~/rubyhub
+    docker:
+      - image: circleci/ruby:2.6.3-node-browsers
+        environment:
+          RAILS_ENV: test
+          PGHOST: 127.0.0.1
+      - image: circleci/postgres:10.10
+        environment:
+          POSTGRES_USER: circleci
+    environment:
+      CC_TEST_REPORTER_ID: e8ca212e976015d9bf1c9a519376032b4919c7da46a7cc30fd14de2eff2a5945
+
+commands:
+  defaults:
+    steps:
+      - checkout
+
+      - save_cache:
+          paths:
+            - /vendor/bundle
+          key: v1-dependencies-{{ checksum "rubyhub.gemspec" }}
+
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "rubyhub.gemspec" }}
+            - v1-dependencies-
+
+      - run:
+          name: install dependencies
+          command: |
+            gem update --system
+            gem install bundler
+            bundle install --jobs=4 --retry=3 --path vendor/bundle
+
+      - run:
+          name: Install Code Climate Test Reporter
+          command: |
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            chmod +x ./cc-test-reporter
+
+  run_linters:
+    steps:
+      - run:
+          name: Rubocop
+          command: bundle exec rubocop
+
+      - run:
+          name: Fasterer
+          command: bundle exec fasterer
+
+      - run:
+          name: run tests
```

Comment:
Job called `run_linters`, but here is specs run.. Linters and specs should be separated jobs.